homeassistant/components/geniushub/config_flow.py

```diff
@@ -0,0 +1,205 @@
+"""Config flow for Geniushub logger integration."""
+
+from __future__ import annotations
+
+from http import HTTPStatus
+import logging
+import socket
+from typing import Any
+
+import aiohttp
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.config_entries import ConfigFlowResult
+from homeassistant.const import (
+    CONF_HOST,
+    CONF_MAC,
+    CONF_PASSWORD,
+    CONF_TOKEN,
+    CONF_USERNAME,
+)
+from homeassistant.helpers import config_validation as cv
+
+from . import validate_input
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+Cloud_API_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_TOKEN): cv.string,
+        vol.Required(CONF_MAC): cv.string,  # vol.Match(MAC_ADDRESS_REGEXP),
```

Comment:
If I check the validate_input function correctly, we don't use the mac. So why do we ask the user for it? (Note: this is not a continuation of the other discussion we had, just a new remark)